src/GherkinCompatibilityMode.php

```diff
@@ -35,14 +35,25 @@ public function shouldRemoveStepKeywordSpace(): bool
     /**
      * @internal
      */
-    public function shouldRemoveFeatureDescriptionPadding(): bool
+    public function shouldRemoveDescriptionPadding(): bool
```

Comment:
Renamed because in legacy mode, padding is also removed from descriptions that get combined into `title` on elements that did not previously support a description.